custom_components/tesla_custom/number.py

```
@@ -122,8 +122,13 @@ class TeslaEnergyBackupReserve(TeslaEnergyEntity, NumberEntity):
 
     async def async_set_native_value(self, value: int) -> None:
         """Update backup reserve percentage."""
-        await self._energysite.set_reserve_percent(value)
-        self.async_write_ha_state()
+        try:
+            await self._energysite.set_reserve_percent(value)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)